icu4c/source/common/locid.cpp

```diff
@@ -1030,7 +1030,8 @@ Locale::forLanguageTag(StringPiece tag, UErrorCode& status)
     // uloc_forLanguageTag() to parse it if it somehow is able to detect that
     // the string actually is BCP-47. This works well for things like strings
     // using BCP-47 extensions, but it does not at all work for things like
-    // BCP-47 grandfathered tags (eg. "en-GB-oed") which are possible to also
+    // legacy language tags (marked as “Type: grandfathered” in BCP 47, e.g., "en-GB-oed")
```

Comment:
Note that the lines directly before and after this line write "BCP-47" instead of "BCP 47".

Reply:
Fixed. https://www.rfc-editor.org/info/bcp47 uses a space, not a dash.